porch/api/porchconfig/v1alpha1/config.porch.kpt.dev_repositories.yaml

```diff
@@ -42,6 +42,9 @@ spec:
     - jsonPath: .status.conditions[?(@.type=='Ready')].status
       name: Ready
       type: string
+    - jsonPath: .spec['git','oci']['repo','registry']
```

Comment:
We _could_ change the API and instead of `repo` and `registry` for Git and OCI respectively, use Address.

It would create inconsistency with Kptfile design though (https://github.com/GoogleContainerTools/kpt/blob/main/pkg/api/kptfile/v1/types.go#L132) so I would not recommend going that route.